### PR TITLE
[0.9] RHUI client certificate refresh for RHEL 7-RAW images

### PR DESCRIPTION
**Pull request by plirglo**

### CODE REVIEW

**Review by seriva — COMMENTED** [review]
How does this effect AWS or offline installations as this seems universal?

**Comment by przemyslavic** [comment]
I am going to test it before merging.

Edit: It fails for AWS.
```
"Executing: yum update -y --disablerepo=* --enablerepo=*microsoft*", "Loaded plugins: amazon-id, search-disabled-repos", "There are no enabled repos.", " Run \"yum repolist all\" to see the repos you have.", " To enable Red Hat Subscription Management repositories:", "     subscription-manager repos --enable <repo>", " To enable custom repositories:", "     yum-config-manager --enable <repo>", "ERROR: Command failed: yum update -y --disablerepo=* --enablerepo=*microsoft*"]}
```

File: CHANGELOG-0.9.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ### Fixed
 
 - [#2262](https://github.com/epiphany-platform/epiphany/issues/2262) - [Ubuntu] elasticsearch-curator in version 5.8.3 is not available from APT repo
+- [#2312](https://github.com/epiphany-platform/epiphany/issues/2312) - [RedHat/Azure] RHUI client certificate expired for RHEL 7-RAW images
```
Comment (to-bar) [inline CHANGELOG-0.9.md]:
```suggestion
- [#2312](https://github.com/epiphany-platform/epiphany/issues/2312) - [RHEL/Azure] RHUI client certificate expired for RHEL 7-RAW images
```

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/download-requirements.sh [hunk 2]
```diff
@@ -503,6 +503,12 @@ else
 	fi
 fi
 
+# Fix for RHUI client certificate expiration [Epiphany issue #2312]
+
+if is_repo_enabled "microsoft"; then
+    yum update -y --disablerepo='*' --enablerepo='*microsoft*'
+fi
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/download-requirements.sh]:
```suggestion
# Update RHUI client package to update expired certificate [Epiphany issue #2312]
if is_repo_enabled "rhui-microsoft-azure-rhel"; then
    yum update -y --disablerepo='*' --enablerepo='rhui-microsoft-azure-rhel*'
fi
```

**Review by to-bar — CHANGES_REQUESTED** [review]

**Review by to-bar — APPROVED** [review]

**Review by przemyslavic — APPROVED** [review]

**Review by seriva — APPROVED** [review]

**Pull request merged**